src/app/search/power-levels.ts

```diff
@@ -0,0 +1,10 @@
+import { D2CalculatedSeason, D2SeasonInfo } from 'data/d2/d2-season-info';
+
+// shortcuts for power numbers
+const currentSeason = D2SeasonInfo[D2CalculatedSeason];
+export const powerLevelByKeyword = {
```

Comment:
This is moved out to avoid a transitive dependency on `d2-season-info` from things that don't need power level info.